src/main/java/core/basesyntax/model/Cat.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.model;
+
+public class Cat {
+    private int age;
+    private String name;
+    private String ownerName;
+
+    @Override
+    public String toString() {
+        return "Cat{"
+                +
+                "age="
+                + age
+                +
+                ", name='"
+                + name + '\''
+                +
+                ", ownerName='"
+                + ownerName + '\''
+                +
+                '}';
```

Comment:
Variable name should be on the same line with variable value for better readability.